### PR TITLE
refactor: APPS-1988 standardize banner styling on all templates

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-1988](https://jira.library.ucla.edu/browse/APPS-1988)

### CODE REVIEW

File: pages/about/programs/_slug.vue [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
+                :text="page.text"
+                :email="page.email"
+                :phone="page.phoneNumber"
+                :staff-directory-link="parsedStaffDirectory"
+                :button-text="parsedButtonText"
+                :to="parsedButtonTo"
+            />
+        </section-wrapper>
 
         <banner-header
```
Comment (dianneweinthal) [inline pages/about/programs/_slug.vue:22]: swap banner-header with banner-text in the section-wrapper

Comment (jendiamond) [reply pages/about/programs/_slug.vue]:
```html
       <banner-text
            v-if="!page.heroImage || page.heroImage.length == 0"
            :title="page.title"
            :text="page.text"
            :email="page.email"
            :phone="page.phoneNumber"
            :staff-directory-link="parsedStaffDirectory"
            :button-text="parsedButtonText"
            :to="parsedButtonTo"
        />

        <section-wrapper class="section-banner">
            <banner-header
                v-if="page.heroImage && page.heroImage.length == 1"
                :image="page.heroImage[0].image[0]"
                :title="page.title"
                :text="page.text"
                :email="page.email"
                :phone="page.phoneNumber"
                :staff-directory-link="parsedStaffDirectory"
                :prompt="parsedButtonText"
                :to="parsedButtonTo"
                :align-right="true"
            />
        </section-wrapper>
```

**Review by pghorpade — DISMISSED** [review]
👍

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**